WordPress/gutenberg
# Surface: Convert component to TypeScript

## What?
Converts the `Surface` component to TypeScript.

## Why?
Part of the @wordpress/components's TypeScript migration (#35744).

## How?
Refactors the current `Surface` component to TypeScript.

## Testing Instructions
- Run npm run dev and ensure no typing errors
- Fire up the Storybook and confirm the `Surface` continues to function as expected
